ui/app/mainui/SendModalHandler.qml

```diff
@@ -545,6 +549,34 @@ QtObject {
                     return ""
                 }
 
+                property int indexOfTxPathUnderReview: -1
+                readonly property bool reviewingLastTxPath: !!handler.fetchedPathModel && handler.indexOfTxPathUnderReview === handler.fetchedPathModel.rowCount() - 1
+                readonly property var txPathUnderReview: {
```

Comment:
@saledjenic we would have to use ModelEntry instead as we are storing the instance, this may lead to a crash in case the model turns null

Reply:
@Khushboo-dev-cpp could you tell me what you mean by that? An example?
Haven't used ModelEntry ever.

Reply:
sure

```
    ModelEntry {
        id: txPathUnderReviewEntry

        sourceModel: handler.fetchedPathModel
        key: "index"
        value: handler.indexOfTxPathUnderReview
    }
```
    
    and to use something like `txPathUnderReview.item.available && !!txPathUnderReview.item`
    
    
   something that we dont have is `index` as a rolename in the fetchedPathModel and we need to add that somehow, I am thinking how to achieve that